python/ray/_private/node.py

```diff
@@ -405,17 +412,14 @@ def check_version_info(self):
         Raises:
             Exception: An exception is raised if there is a version mismatch.
         """
-        import ray._private.usage.usage_lib as ray_usage_lib
-
-        cluster_metadata = ray_usage_lib.get_cluster_metadata(self.get_gcs_client())
-        if cluster_metadata is None:
-            cluster_metadata = ray_usage_lib.get_cluster_metadata(self.get_gcs_client())
+        if not self._cluster_metadata:
+            self._cluster_metadata = self._read_cluster_info_from_kv()
```

Comment:
`cluster_metadata` is reused in `check_version_info(self)` to keep the kv store calling count unchanged.